src/wizards/audience/views/donations/configuration/index.tsx

```diff
@@ -253,8 +253,43 @@ const Donation = () => {
 			auxData: { saveDonationProduct: true },
 		} );
 
+	// Check for product validation errors.
+	const validationResults = Object.values( wizardData.product_validation || {} );
+	const hasInvalidProducts = validationResults.every( product => product.issues.length !== 0 );
```

Comment:
I had to change this in order to get the issues to display:

```suggestion
	const hasInvalidProducts = validationResults.some( product => product.issues.length > 0 );
```